impeller/renderer/backend/vulkan/fence_waiter_vk.cc

```diff
@@ -92,85 +90,116 @@ void FenceWaiterVK::Main() {
   using namespace std::literals::chrono_literals;
 
   while (true) {
-    std::unique_lock lock(wait_set_mutex_);
+    bool terminate;
```

Comment:
I have an uninitialized vars phobia. Can you default it to `false` even if its redundant?

Reply:
I would say that's consistent with https://google.github.io/styleguide/cppguide.html#Local_Variables as well. Done.